middleware.go

```diff
@@ -24,6 +24,9 @@ import (
 
 	"github.com/ant0ine/go-json-rest/rest"
 	"github.com/mendersoftware/deployments/config"
+	"github.com/mendersoftware/deployments/utils/accesslog"
```

Comment:
up to @maciejmrowiec 

it's not clear to me what's the status of our shared library. I remember asking a couple weeks ago if there's a repo for it, and found out that we're not doing this after all? can't recall if it was jira/github/slack.